jfrog_platform/datadog_checks/jfrog_platform/check.py

```diff
@@ -0,0 +1,47 @@
+from datadog_checks.base import ConfigurationError, OpenMetricsBaseCheck
+
+
+class JfrogPlatformCheck(OpenMetricsBaseCheck):
+    """
+    Collect metrics from JFrog
+    """
+
+    def __init__(self, name, init_config, instances=None):
+
+        instance = instances[0]
+        instancetype = instance.get('instance_type')
+        if instancetype == 'artifactory':
+            endpoint = instance.get('prometheus_url')
+            if endpoint is None:
+                raise ConfigurationError("Unable to find prometheus_url in config file.")
+
+            instance.update(
+                {
+                    'prometheus_url': endpoint,
+                    'namespace': 'jfrog.artifactory',
+                    'metrics': ['sys*', 'jfrt*', 'app*'],
+                    'send_distribution_counts_as_monotonic': instance.get(
+                        'send_distribution_counts_as_monotonic', True
+                    ),
+                    'send_distribution_sums_as_monotonic': instance.get('send_distribution_sums_as_monotonic', True),
+                }
+            )
+
+        if instancetype == 'xray':
+            endpoint = instance.get('prometheus_url')
+            if endpoint is None:
+                raise ConfigurationError("Unable to find prometheus_url in config file.")
+
+            instance.update(
+                {
+                    'prometheus_url': endpoint,
+                    'namespace': 'jfrog.xray',
+                    'metrics': ['app*', 'db*', 'go*', 'queue*', 'sys*', 'jfxr*'],
+                    'send_distribution_counts_as_monotonic': instance.get(
+                        'send_distribution_counts_as_monotonic', True
+                    ),
+                    'send_distribution_sums_as_monotonic': instance.get('send_distribution_sums_as_monotonic', True),
+                }
+            )
+
```

Comment:
Let's add an `else` case to make the integration construction fail in case `instance_type` is not recognized